demo/scripts/controls/sidePane/editorOptions/ExperimentalFeatures.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import BuildInPluginState from '../../BuildInPluginState';
+import { ExperimentalFeatures } from 'roosterjs-editor-types';
+
+export interface ExperimentalFeaturesProps {
+    state: ExperimentalFeatures[];
+    resetState: (callback: (state: BuildInPluginState) => void, resetEditor: boolean) => void;
+}
+
+const FeatureName = {
+    [ExperimentalFeatures.ListChain]: 'Enable List Chain for numbering list',
+    [ExperimentalFeatures.MergePastedLine]:
+        'When paste, try merge pasted content to the same line with existing content',
+    [ExperimentalFeatures.NewBullet]: 'Toggle bullet using VList',
+    [ExperimentalFeatures.NewIndentation]: 'Toggle indentation using VList',
+    [ExperimentalFeatures.NewNumbering]: 'Toggle numbering using VList',
+};
+
+export default class ExperimentalFeaturesPane extends React.Component<
+    ExperimentalFeaturesProps,
+    {}
+> {
+    render() {
+        return (
+            <>
+                {[
+                    ExperimentalFeatures.ListChain,
+                    ExperimentalFeatures.MergePastedLine,
+                    ExperimentalFeatures.NewBullet,
+                    ExperimentalFeatures.NewIndentation,
+                    ExperimentalFeatures.NewNumbering,
+                ].map(name => this.renderFeature(name))}
+            </>
+        );
+    }
+
+    private renderFeature(name: ExperimentalFeatures): JSX.Element {
+        let checked = this.props.state.indexOf(name) >= 0;
+        return (
+            <div>
```

Comment:
Since this div has no styling, is it required that these elements be placed in a div vs using fragments?

Reply:
No. This DIV is for line break. Without it all items will be in the same line